HACKING.md

```diff
@@ -138,55 +140,87 @@ If a test hangs, you can enable verbose mode:
 
 (or -check.v for less verbose output).
 
-There is more to read about the testing framework on the [website](https://labix.org/gocheck)
+There is more to read about the testing framework on the 
+[website](https://labix.org/gocheck)
 
-### Running the spread tests
+### Running spread tests
 
-To run the spread tests locally you need the latest version of spread
-from https://github.com/snapcore/spread. It can be installed via:
+To run the spread tests locally via QEMU, you need the latest version of
+[spread](https://github.com/snapcore/spread). You can get spread, QEMU, and the
+build tools to build QEMU images with:
 
-    $ sudo apt install qemu-kvm autopkgtest
-    $ sudo snap install --devmode spread
+    $ sudo apt update && sudo apt install -y qemu-kvm autopkgtest
+    $ curl https://niemeyer.s3.amazonaws.com/spread-amd64.tar.gz | tar -xz -C $GOPATH/bin
 
-Then setup the environment via:
+#### Building spread VM images
 
-    $ mkdir -p .spread/qemu
-    $ cd .spread/qemu
-    # For xenial (same works for yakkety/zesty)
-    $ adt-buildvm-ubuntu-cloud -r xenial
-    $ mv adt-xenial-amd64-cloud.img ubuntu-16.04.img
-    # For trusty
-    $ adt-buildvm-ubuntu-cloud -r trusty --post-command='sudo apt-get install -y --install-recommends linux-generic-lts-xenial && update-grub'
+To run the spread tests via QEMU you need to create VM images in the
+`~/.spread/qemu` directory:
+
+    $ mkdir -p ~/.spread/qemu
+    $ cd ~/.spread/qemu
+
+Assuming you are building on an Ubuntu 18.04 Bionic Beaver (or a later 
```

Comment:
~~an~~, also we normally write Ubuntu 18.04.2 LTS (Bionic Beaver) with the release name in brackets - same for Disco Dingo.
ie.
`Assuming you are building on Ubuntu 18.04 LTS (Bionic Beaver), or a later release like Ubuntu 19.04 (Disco Dingo), run the following to build a 64-bit Ubuntu 16.04 VM to run the spread test on:`

Reply:
If you are not on those systems, pointing developers to spread.zygoon.pl could be an alternative.

Reply:
Okay, I changed all instances to match that pattern, but do we need to add the .2 ? At some point there will be an 18.04.3, and 18.04.4, etc and it seems clear that just saying 18.04 refers to all of those. Also, for clarity dev releases like Ubuntu 19.04 (Disco Dingo) don't have LTS in them, but do they have a "Dev" name in there somewhere?

Reply:
I think it's ok just to mark the LTS releases as I think readers will be used to seeing *LTS* and may question their memory if its missing. I normally only mention the distro name first time (eg. Bionic Beaver) within a text, and only then if the text exists in relative isolation to other docs (as this does). It's mainly to reduce the mental friction of the same distros being referred to in lots of different ways.

Reply:
Okay, in that case I'll probably drop the duplicated distro names